keyboards/handwired/3dp660_oled/info.json

```
@@ -0,0 +1,246 @@
+{
+    "manufacturer": "xia0",
+    "keyboard_name": "handwired/3dp660_oled",
+    "maintainer": "xia0",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
```

migrated from `rules.mk`
```suggestion
        "mousekey": true,
        "oled": true,
        "nkro": true,
        "wpm": true
    },
    "build": {
        "lto": true
    },
```